data/learning.yml

```diff
@@ -22,6 +22,11 @@ books:
       - link: /learn/quickstart/organising_code
       - link: /learn/quickstart/derived_types
 
+  - title: Floating Point Numbers comparison
+  description: Learn how to handle rounding errors in floating-point numbers, use them in if and for loops, and make robust comparisons using tolerance in Fortran.
+  category: Getting started
+  link: /learn/Floating_Point_Numbers_comparison
```

Comment:
> The Floating Point Number section of **Fortran Best Practices** could be extended with 2 subsections:

As described by @band-a-prend, "Floating Point Numbers comparison" is recommended to be merged into the link `/learn/best_practices/floating_point` in Fortran Best Practices.